### PR TITLE
supress noisy logs on startup in dev profile

### LINKED ISSUE #13835: DEBUG  [  restartedMain] Validator   : ERROR [ModelSpecification.spec] At least one type of specification is required
##### **Overview of the issue**

when start up get several debug error log

2021-02-04 08:47:51.637 DEBUG 10814 --- [  restartedMain] Validator                                : ERROR [ModelSpecification.spec] At least one type of specification is required

##### **JHipster Version(s)**

latest 7.0.0 Beta.1

##### **Browsers and Operating System**

- [ ] Checking this box is mandatory (this is just to show you read everything)

### LINKED ISSUE #14444: Remove noisy logs in dev profile
##### **Overview of the feature request**

With JH7 the dev profile log have become quite noisy. There are many debug logs now. lets audit and change levels

Some samples

```
021-03-24 17:01:15.182 DEBUG 1644297 --- [  restartedMain] c.ehcache.core // lot of these
2021-03-24 17:01:21.859 DEBUG 1644297 --- [  restartedMain] _.s.web.servlet.HandlerMapping.Mappings // lot of these
2021-03-24 17:01:24.848 DEBUG 1644297 --- [  restartedMain] Validator // lot of these
```

### PR DESCRIPTION
**Pull request by atomfrede**
close #13835
closes #14444 

It now looks like this:

image

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by atomfrede** [comment]
The failing test is a heap space error, pretty sure it is not related to the change :)

**Comment by atomfrede** [comment]
In fact its this error:

image

Seems like this is failing for a bit longer. Is anyone aware of a change with regards to jackson or so?

File: generators/server/templates/src/main/resources/logback-spring.xml.ejs [hunk 1]
```diff
@@ -152,6 +152,10 @@
     <%_ if (cacheProvider === 'infinispan') { _%>
     <logger name="unknown.jul.logger" level="WARN"/>
     <%_ } _%>
+    <!-- See https://github.com/jhipster/generator-jhipster/issues/13835 -->
+    <logger name="Validator" level="INFO"/>
+    <!-- See https://github.com/jhipster/generator-jhipster/issues/14444 -->
+    <logger name="_org.springframework.web.servlet.HandlerMapping.Mappings" level="INFO"/>
```
Comment (pascalgrimaud) [inline generators/server/templates/src/main/resources/logback-spring.xml.ejs:158]: Typo with additional _ ?

Comment (atomfrede) [reply generators/server/templates/src/main/resources/logback-spring.xml.ejs:158]: No in fact not. It's a hidden logger that is prefixed with _

**Comment by atomfrede** [comment]
Going to merge this to we have this fixed for the next release, which seems to be quite near.

**Comment by atomfrede** [comment]
Bounty claimed https://opencollective.com/generator-jhipster/expenses/41604

File: generators/server/templates/src/main/resources/logback-spring.xml.ejs (hunk 1 above)
Comment (deepu105) [reply generators/server/templates/src/main/resources/logback-spring.xml.ejs:158]: May be lets add a comment so no one thinks its a typo 😂

**Pull request merged**